src/Uno.Foundation/Collections/ValueSet.cs

```diff
@@ -10,7 +11,40 @@ namespace Windows.Foundation.Collections
 {
 	public sealed partial class ValueSet : IPropertySet, IObservableMap<string, object>, IDictionary<string, object>, IEnumerable<KeyValuePair<string, object>>
 	{
-		
+		private Dictionary<string, object> _dictionary;
+
+		public ValueSet()
+		{
+			_dictionary = new Dictionary<string, object>();
+		}
+
+		public void Add(string key, object value) => _dictionary.Add(key, value);
```

Comment:
I wonder, does the windows API accept a null `value` parameter? If so, we will need to alter the API here for the nullable references to work properly.

Reply:
Yes, both `Dictionary` and `ValueSet` allows `Add("dummy", null)`

Reply:
Ok, then we'll need to change the definition of this type to accept nullable object:

```csharp
public sealed partial class ValueSet : IPropertySet, IObservableMap<string, object?>, IDictionary<string, object?>, IEnumerable<KeyValuePair<string, object?>>
```

And the rest of the definition that comes with it.

Reply:
ok.